sycl/include/sycl/ext/oneapi/experimental/annotated_ptr/annotated_ptr.hpp

```diff
@@ -71,6 +71,7 @@ class annotated_ref {
 template <typename T, typename... Props>
 class annotated_ref<T, detail::properties_t<Props...>> {
   using property_list_t = detail::properties_t<Props...>;
+  using T_Stripped = typename std::remove_volatile<T>::type;
```

Comment:
This alias is unlike any other. Normally we would call something like `T` the `value_type` of a class when made a public alias, so we might as well follow that, even if this is private. So I would prefer something like:
```suggestion
  using nonvolatile_value_type = std::remove_volatile_t<T>;
```
Or you can just inline it as the alias isn't much more readable at this point.